pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -273,7 +273,7 @@ def _register_ddp_hooks(self) -> None:
             _TORCH_GREATER_EQUAL_1_8 and self.on_gpu and self._is_single_process_single_device
         ):
             register_ddp_comm_hook(
-                model=self._model,
+                model=self.model,
```

Comment:
May I suggest you also remove the getter/setter in the TTP and rename the init attribute `self._model -> self.model`? No subclass overrides it.

Reply:
I suggest to keep it for the purpose of documentation

Reply:
But the class docstring can define its attributes, see for example the `Loop` docstring